caffe2/core/macros.h.in

```diff
@@ -43,7 +43,7 @@ static_assert(
 #cmakedefine CAFFE2_USE_MKLDNN
 #cmakedefine CAFFE2_USE_NVTX
 #cmakedefine CAFFE2_USE_TRT
-#cmakedefine CAFFE2_DISABLE_NUMA
+#cmakedefine C10_DISABLE_NUMA
```

Comment:
wait, you probably want to move necessary flags here - https://github.com/pytorch/pytorch/blob/master/c10/macros/cmake_macros.h.in

and please, please test that CMake flags actually work as expected :-)

Reply:
Yup - you should really move this.